ci/export_lib_versions.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Read `lib-versions.env` file line by line and export every env variable
+# defined in that file only if the env variable with such name does not exist
+# yet. This allows overriding the variables by the pipeline, but there is always
+# default defined in one place - `lib-versions.env`.
+while read -r line || [[ -n "$line" ]]; do
+  # if the line is not a comment and is not empty; then
+  if ! [[ "$line" =~ ^#.*$ ]] && [[ -n "$line" ]]; then
+    key=$(echo "${line}" | cut -d'=' -f1)
+    echo "key = ${key}"
+    # only export the variable if it is not already set
+    if [[ -z "${!key:-}" ]]; then
+      export "${line?}"
```

Comment:
[minor] why need `?` - variable `line` is checked on line #10 already

Reply:
It's just to silence the shellcheck. It gives a warning, because it assumes that what I want is `export line`
But I'll silence the warning with explicit comment, it's less confusing

Reply:
Addressed here https://github.com/NordSecurity/nordvpn-linux/pull/575/files#diff-8ab3b1bcf4c95085b64d3a617e07c57db959710da54d8743a531b31760dc2f5aR15